homeassistant/components/hydrawise/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/hydrawise",
   "iot_class": "cloud_polling",
   "loggers": ["pydrawise"],
-  "requirements": ["pydrawise==2023.11.0"]
+  "requirements": ["pydrawise==2023.12.1"]
```

Comment:
Is the legacy lib still in this version? If so, please bump the lib in a separate PR

Reply:
https://github.com/home-assistant/core/pull/107032